Source/Upstream: Reported, https://gitlab.gnome.org/GNOME/gthumb/merge_requests/3
Reason: Fix libdir 

From 7d5bae28895e3300b99f995d6f20ebada60cbf81 Mon Sep 17 00:00:00 2001
From: Kalev Lember <klember@redhat.com>
Date: Tue, 19 Feb 2019 16:57:30 +0100
Subject: [PATCH] Use meson libdir for extensions instead of hardcoding 'lib'

This fixes the build on distros that use /usr/lib64, e.g. Fedora.
---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index 9edf700c..4953835a 100644
--- a/meson.build
+++ b/meson.build
@@ -33,7 +33,7 @@ prefix = get_option('prefix')
 datadir = join_paths(prefix, get_option('datadir'))
 pkgdatadir = join_paths(datadir, meson.project_name())
 pkgincludedir = join_paths(prefix, 'include', meson.project_name())
-pkglibdir = join_paths(prefix, 'lib', meson.project_name())
+pkglibdir = join_paths(prefix, get_option('libdir'), meson.project_name())
 po_dir = join_paths(meson.source_root(), 'po')
 extensions_install_dir = join_paths(pkglibdir, 'extensions')
 ui_install_dir = join_paths(pkgdatadir, 'ui')
-- 
2.18.1

